### ISSUE TITLE
Cannot convert undefined or null to object

### ISSUE THREAD

**Issue by miljan-aleksic** [AtomLinter/linter-eslint#1023]
## Issue Type
Bug

## Issue Description
Just after installing the new version. I tried updating eslint to 4.7.0, was using 4.5.0. Restarting, reinstalling, nothing seems to fix it.

```js
Cannot convert undefined or null to object TypeError: Cannot convert undefined or null to object
    at hasOwnProperty (<anonymous>)
    at /Users/miljan/.atom/packages/linter-eslint/src/worker.js:27:41
    at Map.forEach (native)
    at updateFixableRules (/Users/miljan/.atom/packages/linter-eslint/src/worker.js:26:21)
    at lintJob (/Users/miljan/.atom/packages/linter-eslint/src/worker.js:61:3)
    at process.<anonymous> (/Users/miljan/.atom/packages/linter-eslint/src/worker.js:101:22)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:191:7)
    at process.nextTick (internal/child_process.js:752:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
    at process._tickCallback (internal/process/next_tick.js:98:9)
```

## Bug Checklist
- [x] Restart Atom
- [x] Verify the `eslint` CLI gives the proper result, while `linter-eslint` does not
- [x] Paste the output of the `Linter Eslint: Debug` command from the Command Palette below

```
Atom version: 1.20.0
linter-eslint version: 8.3.0
ESLint version: 4.7.0
Hours since last Atom restart: 0.1
Platform: darwin
Using local project ESLint from: /Users/miljan/repos/@zoolanders/zooadmin-ui/node_modules/eslint
Current file's scopes: [
  "source.js.jsx",
  "keyword.control.module.js"
]
linter-eslint configuration: {
  "lintHtmlFiles": true,
  "useGlobalEslint": false,
  "showRuleIdInMessage": true,
  "disableWhenNoEslintConfig": true,
  "eslintrcPath": "",
  "globalNodePath": "",
  "advancedLocalNodeModules": "",
  "eslintRulesDir": "",
  "disableEslintIgnore": false,
  "disableFSCache": false,
  "fixOnSave": false,
  "scopes": [
    "source.js",
    "source.jsx",
    "source.js.jsx",
    "source.babel",
    "source.js-semantic"
  ],
  "rulesToSilenceWhileTyping": [],
  "rulesToDisableWhileFixing": [],
  "ignoreFixableRulesWhileTyping": false
}
```

**Comment by Arcanemagus** [comment]
Hmmmm, I have a theory as to what might be causing this, are you able to share a way for me to reproduce this and test?

**Comment by sr1994lu** [comment]
I did updated linter-eslint to 8.3.1. However same issue.

**Comment by Arcanemagus** [comment]
@sr1994lu Are you able to share a project that I can try to reproduce this on? I have a guess what might be wrong but I have no way to confirm this as I haven't seen it myself.

**Comment by sr1994lu** [comment]
@Arcanemagus this is [my repo](https://github.com/sr1994lu/for_you)

**Comment by DerTieran** [comment]
For me adding the [here](https://github.com/AtomLinter/linter-eslint/blob/master/src/worker.js#L32) fixes it:
```js
if (props.meta && Object.prototype.hasOwnProperty.call(props.meta, 'fixable')) {
  currentRules.add(rule)
}
```
The problem was that the props object was empty.

**Comment by Arcanemagus** [comment]
Thanks for the confirmation @DerTieran, that's what I was guessing the issue might be 😉.

**Comment by DerTieran** [comment]
@Arcanemagus no problem was a bit to late to see your commit 😄 

FYI you can reproduce it with plugins like [`eslint-plugin-promise`](https://github.com/xjamundx/eslint-plugin-promise).

**Comment by Arcanemagus** [comment]
Fixed in v8.3.2, thanks for the reports everyone and sorry for the trouble!
